react-native/react/tabs/devices/index.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+/* @flow */
+
+import React from '../../base-react'
+import BaseComponent from '../../base-component'
+
+import CodePage from '../../login2/register/code-page'
+import GenPaperKey from './gen-paper-key'
+import ExistingDevice from '../../login2/register/existing-device'
+import RemoveDevice from './remove-device'
+
+import { loadDevices } from '../../actions/devices'
+import { routeAppend } from '../../actions/router'
+import DevicesRender from './devices-render'
+
+export default class Devices extends BaseComponent {
+  constructor (props) {
+    super(props)
+  }
```

Comment:
@Sidnicious Hm, which part?  I think every component is supposed to call super().